### PR TITLE
Add generator for Indonesian language

### PR DESCRIPTION
**Pull request by agasigp**

### CODE REVIEW

**Comment by jzonta** [comment]
@agasigp You changed en_US language, added language Indonesian in en_US. Please check your commit.

**Comment by agasigp** [comment]
Ups, my bad. I just fixed that.

File: src/Provider/en_US/Restaurant.php [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 class Restaurant extends \Faker\Provider\Base
 {
-
```
Comment (jzonta) [inline src/Provider/en_US/Restaurant.php:7]: The file did not have significant changes, it is not necessary to be sent, you must remove the modified line.

**Review by jzonta — CHANGES_REQUESTED** [review]
The file  "src/Provider/en_US/Restaurant.php"  did not have significant changes, it is not necessary to be sent, you must remove the modified line.

New names added. Could you do the translation?

**Comment by jzonta** [comment]
Close because you did not respond to the change request and another user submitted an equal PR

**Pull request closed without merging**